tests/integration/synapseutils/test_synapseutils_sync.py

```diff
@@ -5,32 +5,47 @@
 import datetime
 from func_timeout import FunctionTimedOut, func_set_timeout
 import pandas as pd
-import numpy as np
 
 import pytest
 
 from synapseclient.core.exceptions import SynapseHTTPError
-from synapseclient import Entity, File, Folder, Link, Project, Schema, Synapse
+from synapseclient import (
+    File,
+    Folder as SynapseFolder,
+    Link,
+    Project as SynapseProject,
+    Schema,
+    Synapse,
+)
+from synapseclient.models import Project, Folder
 import synapseclient.core.utils as utils
 import synapseutils
 
 from tests.integration import QUERY_TIMEOUT_SEC
 
+BOGUS_ACTIVITY = "bogus_activity"
+BOGUS_DESCRIPTION = "bogus_description"
+SYNAPSE_URL = "https://www.synapse.org"
+SUB_SYNAPSE_URL = "https://www.asdf.synapse.org"
+SEND_MESSAGE = False
+
 
 @pytest.mark.asyncio(scope="session")
 @pytest.fixture(scope="function", autouse=True)
 async def test_state(syn: Synapse, schedule_for_cleanup):
     class TestState:
         def __init__(self):
             self.syn = syn
-            self.project = syn.store(Project(name=str(uuid.uuid4())))
-            self.folder = syn.store(Folder(name=str(uuid.uuid4()), parent=self.project))
+            self.project = syn.store(SynapseProject(name=str(uuid.uuid4())))
+            self.folder = syn.store(
```

Comment:
Nit: Is this going to be a pain to switch out when we move all the objects?

Reply:
It should not be. Anything left using this is functionality left in sync.py that'll end up changing, and we will update the related integration tests at that point. Or it's code that'll be deprecated and removed.